src/pages/HomePage/HomePage.tsx

```diff
@@ -15,7 +18,14 @@ const HomePage: React.FC = () => {
         </ThemedToolbar>
       </IonHeader>
       <IonContent>
-        <StudyList />
+        {isLoggedIn ? (
+          <StudyList />
+        ) : (
+          <div slot="fixed">
+            You must <a onClick={initiateLogin}>log in</a> to view and edit
+            studies
+          </div>
```

Comment:
I think I'd prefer this without the "You must":
```diff
- You must <a onClick={initiateLogin}>log in</a> to view and edit
- studies
+ <a onClick={initiateLogin}>Log in</a> to view and edit studies
```

I don't consider this to be a deal-breaker, though.